### PR TITLE
Add monitor user + roles

### PR DESCRIPTION
**Pull request by MiaAltieri**
## Problem: 
Conversation with Guillermo and Mykola illustrated that a the MongoDB exporter charm couldn't run commands due to authorisation issues 

## Solution:
1. Create a monitor user
2. Create monitor roles

## Testing
Integration tests verify we can run the necessary commands

### CODE REVIEW

File: tests/integration/helpers.py [hunk 1]
```diff
@@ -36,9 +36,11 @@ async def get_password(ops_test: OpsTest, app=APP_NAME) -> str:
     unit_name = ops_test.model.applications[app].units[0].name
     unit_id = unit_name.split("/")[1]
 
-    action = await ops_test.model.units.get(f"{app}/{unit_id}").run_action("get-password")
+    action = await ops_test.model.units.get(f"{app}/{unit_id}").run_action(
+        "get-password", **{"username": username}
+    )
     action = await action.wait()
-    return action.results["operator-password"]
+    return action.results[f"{username}-password"]
```
Comment (delgod) [inline tests/integration/helpers.py:43]: lets simplify parsing of results to 3rd-party charms.
```suggestion
    return action.results["password"]
```
can we check what keys other charms have?

**Review by delgod — CHANGES_REQUESTED** [review]

File: tests/integration/helpers.py (hunk 1 above)
Comment (MiaAltieri) [reply tests/integration/helpers.py:43]: Yes will do. 

[This is the info](https://docs.google.com/spreadsheets/d/1mb_cN92iEP0UEJzh7RHZXwJlYd0Yi_y7N6Frw4z3rUs/edit#gid=0) I've collected on how the other charms manage passwords

**Review by marceloneppel — APPROVED** [review]

File: tests/integration/helpers.py (hunk 1 above)
Comment (marceloneppel) [reply tests/integration/helpers.py]: @delgod, should we align with the other database charm developers to change the other charms as well?

Comment (delgod) [reply tests/integration/helpers.py]: @marceloneppel yes please

**Review by delgod — APPROVED** [review]

**Pull request merged**